### ISSUE TITLE
Error: application 'C:\Program Files (x86)\Eclipse\Sumo\bin\sumo' not found

### ISSUE THREAD

**Issue by transport45678** [eclipse-sumo/sumo#14676]
I downloaded latest development version using msi installer file, my batch file command `python "%SUMO_HOME%/tools/runSeeds.py" -k scenario.sumocfg --seeds 1:11` does not work. It gives an error :Error: application 'C:\Program Files (x86)\Eclipse\Sumo\bin\sumo' not found.
But it works version 1.19.0 (installed again msi installer file).
What is the difference between two version?. Can you help to solve this error? Regards.

**Comment by palvarezlopez** [comment]
@transport45678 the previous version was already installed in a path with spaces? "Program Files (x86)"

**Comment by transport45678** [comment]
For first question: I did not change any spaces for both version while installing. Both version are installed in a path with spaces.
Second question: I uninstalled the previous version and rebooting before installing new one. It does not work again.

Interestingly, before running simulation I am using another command to create route files as: 
`python "%SUMO_HOME%/tools/routeSampler.py" -r initial_routes.rou.xml -o final_routes.rou.xml -f poisson --turn-files turn_count.xml --turn-max-gap 7 --attributes="type=\"typedist1\" departLane=\"best\" departSpeed=\"random\""`

This command works both of version (v1.19.0 and latest development version)

But while running simulation using batch file, I am taking error mentioned in the first post.
